observations/fashion_mnist.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import numpy as np
+
+from observations.util import maybe_download_and_extract
+
+
+def fashion_mnist(path):
+  """Load the Fashion MNIST data set (Xiao et al., 2017).
+
+  Args:
+    path: str.
+      Path to directory which either stores file or otherwise file will
+      be downloaded and extracted there. Filenames are
+      `train-images-idx3-ubyte`, `train-labels-idx1-ubyte`,
+      `t10k-images-idx3-ubyte`, `t10k-labels-idx1-ubyte`.
+
+  Returns:
+    Tuple of np.ndarray's
+    `(x_train, y_train), (x_test, y_test)`.
+  """
+  path = os.path.expanduser(path)
+  url = 'http://fashion-mnist.s3-website.eu-central-1.amazonaws.com'
```

Comment:
Thanks for adding this. This downloading successfully only after appending `/` to the URL.